src/docs/development/platform-integration/platform-views.md

```diff
@@ -196,58 +243,299 @@ public class MainActivity extends FlutterActivity {
         flutterEngine
             .getPlatformViewsController()
             .getRegistry()
-            .registerViewFactory("hybrid-view-type", new NativeViewFactory());
+            .registerViewFactory("<platform-view-type>", new NativeViewFactory());
     }
 }
 ```
 
+For plugin registration, modify the main plugin file (e.g. `PlatformViewPlugin.java`):
+
+```java
+package dev.flutter.plugin.example;
+
+import androidx.annotation.NonNull;
+import io.flutter.embedding.engine.plugins.FlutterPlugin;
+import io.flutter.plugin.common.BinaryMessenger;
+
+public class PlatformViewPlugin implements FlutterPlugin {
+  @Override
+  public void onAttachedToEngine(@NonNull FlutterPluginBinding binding) {
+    binding
+        .getFlutterEngine()
+        .getPlatformViewsController()
+        .getRegistry()
+        .registerViewFactory("<platform-view-type>", new NativeViewFactory());
+  }
+}
+```
+
 For more information, see the API docs for:
 
+* [`FlutterPlugin`][]
 * [`PlatformViewRegistry`][]
 * [`PlatformViewFactory`][]
 * [`PlatformView`][]
 
-
+[`FlutterPlugin`][]: {{site.api}}/javadoc/io/flutter/embedding/engine/plugins/FlutterPlugin.html
 [`PlatformViewFactory`]: {{site.api}}/javadoc/io/flutter/plugin/platform/PlatformViewFactory.html
 [`PlatformViewRegistry`]: {{site.api}}/javadoc/io/flutter/plugin/platform/PlatformViewRegistry.html
 [`PlatformView`]: {{site.api}}/javadoc/io/flutter/plugin/platform/PlatformView.html
 
-Finally, modify your `build.gradle` file to require
-version 19 (or later) of the Android API.
+Finally, modify your `build.gradle` file to require either
+the minimal Android SDK version:
 
 ```gradle
 android {
     defaultConfig {
-        minSdkVersion 19
+        minSdkVersion 19 // if using Hybrid composition.
+        minSdkVersion 20 // if using Virtual display.
     }
 }
 ```
 
-### Performance
+## iOS
 
-Hybrid composition on Android works best on Android 10 or above.
-On lower versions of Android, the Flutter UI can appear slower
-while the native view is rendered on the screen.
-This overhead is due to the synchronization of Flutter frames to the
-Android view system.
+iOS only uses Hybrid composition, which means that the native
+`UIView` is appended to view hierarchy.
 
-To mitigate this issue, we recommend that you avoid displaying
-the native view while an animation is taking place in Dart.
-For example, you could use a placeholder such as capturing a
-screenshot of the native view and rendering the placeholder
-while the animation is happening.
+In Flutter 1.22, platform views are enabled by default. This means
```

Comment:
nit: I'd suggest rephrasing this as "Prior to Flutter 1.22 platform views were in developers preview and setting the... flag was required in order to use them on iOS"

Reply:
Done.